akka-http-core/src/main/scala/akka/http/impl/engine/client/PoolInterfaceActor.scala

```diff
@@ -106,7 +116,7 @@ private class PoolInterfaceActor(hcps: HostConnectionPoolSetup,
         // if we can't dispatch right now we buffer and dispatch when demand from the pool arrives
         if (inputBuffer.isFull) {
           x.responsePromise.failure(
-            new RuntimeException(s"Exceeded configured max-open-requests value of [${inputBuffer.size}]"))
+            new RuntimeException(s"Exceeded configured max-open-requests value of [${inputBuffer.size}]")) // TODO maybe named exception?
```

Comment:
The one in JDK does not take a message, I'll have to roll our own (or use the one from OverflowStrategy)?